src/js/view/view.js

```diff
@@ -76,6 +76,7 @@ function View(_api, _model) {
     let _stateClassRequestId = -1;
 
     let _floatingConfig = _model.get('floating');
+    _floatingConfig = _floatingConfig && _floatingConfig.disabled ? null : _floatingConfig;
```

Comment:
This should be normalized in the config so you don't need to repeat this kind of logic. Either set "floating" to null or always define `floating` with  `disabled = true` so you can depend on it being defined and never have to repeat this `_floatingConfig &&` check.